pkg/util/mount/mount.go

```diff
@@ -130,14 +130,24 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 		}
 	}
 
+	// TODO: this is a workaround for the unmount device issue caused by gci mounter.
```

Comment:
This is a total hack. `gci_mounter` is a GKE specific deployment detail. We should not be leaking that logic into upstream.